recipes/cuda-python/LICENSE

```diff
@@ -0,0 +1,48 @@
+NVIDIA SOFTWARE LICENSE
```

Comment:
With the `license_file` suggestion in the `meta.yaml`, think we should be able to drop this. Should save us from maintaining it in two places in case changes are made to the upstream copy